contributing/giving-constructive-criticism.rst

```diff
@@ -0,0 +1,154 @@
+Giving constructive criticism
+=============================
+
+:doc:`Reviewing issues and pull request </contributing/community/reviews>` is a great way to get started with
+contributing to the Symfony community. Anyone can do it! But before
+you give a comment, take a step back and think, is what you are about
+to say actually what you indent?
+
+Communicating over the internet with nothing but text can pose a
+big challenge, especially if you remember the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+idea's and opinions.
+
+Not everyone speaks English, is able to use a keyboard, and some might
+even have dyslexia. And not to mention that some might have a
```

Comment:
You can remove the `And` and just start `Not to mention that...`

It's bad form to start a sentence with a conjunction

Reply:
Thanks for the tip 👍 TIL.